packages/wdio-config/src/lib/ConfigParser.js

```diff
@@ -266,7 +295,12 @@ export default class ConfigParser {
         let files = []
 
         if (typeof patterns === 'string') {
+            patterns = this.removeLineNumbers(patterns)
             patterns = [patterns]
+        } else {
+            for (let i=0; i< patterns.length; i++) {
+                patterns[i] = this.removeLineNumbers(patterns[i])
+            }
```

Comment:
no need for an else, just do this after the `if`:
```js
patterns = patterns.map(::this.removeLineNumbers)
```

Reply:
Done! :-)